### PR TITLE
Update theme and remove sentry config file

### PR DESCRIPTION
**Pull request by rogermparent**

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -17,27 +17,3 @@ exports.onPostBuild = api => callOnModels(models, 'onPostBuild', api)
...
-
-    const miniCssExtractPlugin = config.plugins.find(
-      plugin => plugin.constructor.name === 'MiniCssExtractPlugin'
-    )
-    if (miniCssExtractPlugin) {
-      miniCssExtractPlugin.options.ignoreOrder = true
-    }
-    actions.replaceWebpackConfig(config)
-  }
-}
```
Comment (rogermparent) [inline gatsby-node.js:43]: Seems like this webpack modification was causing the build to fail

**Comment by rogermparent** [comment]
image
:tada:

**Comment by rogermparent** [comment]
Seems the site itself needs to throw the error to test, and throwing in the console isn't enough.

**Review by yathomasi — APPROVED** [review]
Let's go ahead and see it live.

**Pull request merged**